app/models/user.rb

```diff
@@ -37,4 +41,20 @@ def find_badge_assignment(badge)
   def admin_module?(current_module)
     roles.pluck(:name).include? "admin_#{current_module}"
   end
+
+  private
+  def self.order_ids_by_votes(votes, order)
+    if order == :desc
+      votes.sort_by{|k,v|v}.reverse.to_h.keys
+    else
+      votes.sort_by{|k,v|v}.to_h.keys
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Space missing after comma.<br>Space after closing `|` missing.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.